pages/partials/examQuestionScore.ejs

```diff
@@ -15,3 +16,15 @@
         <span class="badge badge-warning"><%= Math.floor(instance_question.highest_submission_score * 100) %>%</span>
     <% } %>
 <% } %>
+
+<% if ("allow_grade_left_ms" in instance_question && instance_question.allow_grade_left_ms > 0) { %>
+<a class="grade-rate-limit-popover btn btn-xs" data-toggle="popover" data-html="true" data-content="This question limits the rate of submissions. Further grade allowed <%= instance_question.allow_grade_interval %>" data-placement="auto" data-trigger="focus" data-container="body" tabindex="0">
+    <i class="fa fa-hourglass-half" aria-hidden="true"></i>
+</a>
+<script>
+  window.setTimeout(function() {
+    $('.grade-rate-limit-popover').hide();
+  }, <%= instance_question.allow_grade_left_ms %>);
+</script>
```

Comment:
How does this work if there are multiple questions with different timeouts? I think we should either do nothing (that is, remove this script) or do it properly (with unique IDs for each question, etc).

Reply:
Thinking about it more, I think we should just remove this. We aren't going to update the text that says "XX questions are rate limited", so things will get confusing in any case. I suggested a change to the information text to explicitly say that a page reload is needed for updates.

At some point we can rewrite this page with react and have everything update in real time 😄

Reply:
The issue is that this partial is also used in the instance question page, though it's easy enough to add this update in the function that is called when the countdown ends.